mofacts/client/views/home/profile.js

```diff
@@ -427,7 +427,7 @@ async function selectTdf(currentTdfId, lessonName, currentStimuliSetId, ignoreOu
 
   // Check to see if the user has turned on audio prompt.
   // If so and if the tdf has it enabled then turn on, otherwise we won't do anything
-  const userAudioPromptFeedbackToggled = (audioPromptFeedbackView == 'feedback') || (audioPromptFeedbackView == 'all');
+  const userAudioPromptFeedbackToggled = (audioPromptFeedbackView == 'feedback') || (audioPromptFeedbackView == 'all') || (audioPromptFeedbackView == 'question');
```

Comment:
Ideally, this would be a separate commit as it's solving a different issue, though for now this is fine.